### PR TITLE
gas-refund-partioning-process

### PR DESCRIPTION
**Pull request by mwamedacen**

### CODE REVIEW

File: src/service/transaction-fees-indexer/transaction-fees.ts [hunk 1]
```diff
@@ -42,49 +44,72 @@ export async function computeAccumulatedTxFeesByAddress({
...
+  ) {
+    const _endBlock = Math.min(_startBlock + PARTITION_SIZE, endBlock);
+
+    logger.info(
+      `swapTracker start indexing partition between ${_startBlock} and ${
+        _startBlock + PARTITION_SIZE
+      }`,
+    );
+
+    await swapTracker.indexSwaps(_startBlock, Math.min(_endBlock, endBlock));
```
Comment (samthomson) [inline src/service/transaction-fees-indexer/transaction-fees.ts]: isn't this already acieved in `_endBlock`? `Math.min(_endBlock, endBlock)`

File: src/service/transaction-fees-indexer/transaction-fees.ts [hunk 2]
```diff
@@ -42,49 +44,72 @@ export async function computeAccumulatedTxFeesByAddress({
...
+  for (
+    let _startBlock = startBlock;
+    _startBlock < endBlock;
+    _startBlock += PARTITION_SIZE
+  ) {
+    const _endBlock = Math.min(_startBlock + PARTITION_SIZE, endBlock);
+
+    logger.info(
+      `swapTracker start indexing partition between ${_startBlock} and ${
+        _startBlock + PARTITION_SIZE
```
Comment (samthomson) [inline src/service/transaction-fees-indexer/transaction-fees.ts]:
```suggestion
        _startBlock + _endBlock
```

File: src/service/transaction-fees-indexer/transaction-fees.ts (hunk 1 above)
Comment (mwamedacen) [reply src/service/transaction-fees-indexer/transaction-fees.ts]: yes it's an oversight from my local refactoring ^^'

File: src/service/transaction-fees-indexer/transaction-fees.ts (hunk 2 above)
Comment (mwamedacen) [reply src/service/transaction-fees-indexer/transaction-fees.ts]: You probably meant
```diff
-_startBlock + PARTITION_SIZE
+_endblock
```

File: src/service/transaction-fees-indexer/transaction-fees.ts (hunk 1 above)
Comment (mwamedacen) [reply src/service/transaction-fees-indexer/transaction-fees.ts]: fixed here 552e0f547a927185d092d326242d8fd0d9186e13 thx for reporting

File: src/service/transaction-fees-indexer/transaction-fees.ts (hunk 2 above)
Comment (samthomson) [reply src/service/transaction-fees-indexer/transaction-fees.ts]: I did haha

**Review by samthomson — APPROVED** [review]

**Pull request merged**